spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala

```diff
@@ -101,18 +107,20 @@ trait ConformanceExecution extends CommonJobExecution {
     }
   }
 
-  protected def processConformanceResult(args: Array[String],
+  protected def processConformanceResult[T](args: Array[String],
                                          result: DataFrame,
                                          preparationResult: PreparationResult,
                                          menasCredentials: MenasCredentials)
                                         (implicit spark: SparkSession,
-                                         cmd: ConformanceConfig,
+                                         cmd: ConformanceParser[T],
                                          fsUtils: FileSystemVersionUtils): Unit = {
     val cmdLineArgs: String = args.mkString(" ")
 
+    // StandardizationPath is the input on the Conformance phase of the combined job
+    val standardizationPath = preparationResult.pathCfg.standardizationPath.getOrElse(preparationResult.pathCfg.inputPath)
```

Comment:
For any reader of the code not fully the structure of the code, this would be rather confusing... (similar as in `StandardizationExecution`)